pkg/scheduler/eventhandlers_test.go

```diff
@@ -346,3 +349,46 @@ func TestNodeConditionsChanged(t *testing.T) {
 		})
 	}
 }
+
+func TestUpdatePodInCache(t *testing.T) {
+	ttl := 10 * time.Second
+	nodeName := "node"
+
+	tests := []struct {
+		name   string
+		oldObj interface{}
+		newObj interface{}
+	}{
+		{
+			name:   "test update pod with same uid",
+			oldObj: podWithPort("oldUID", nodeName, 80),
+			newObj: podWithPort("oldUID", nodeName, 8080),
```

Comment:
this is not quite accurate test because the pod names are also different, but I think it is fine for the purpose of the test.

Reply:
yes, let me update it.